ui/components/app/auto-detect-token/index.scss

```diff
@@ -0,0 +1,21 @@
+.mm-auto-detect-in-modal {
+  &__benefit {
+    flex: 1;
+  }
+
+  .mm-modal-content {
+    &__dialog {
+      background-position: -80px 16px;
+      background-repeat: no-repeat;
+    }
+  }
+}
+
+.image-default {
+  padding-top: 80%;
+  position: relative;
+  background-image: url('/images/wallet-alpha.png');
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center center;
+}
```

Comment:
We avoid using globally accessible class names and follow the BEM convention when adding styles. Is there a particular reason we're not using an <img /> tag here?